### PR TITLE
collectd: Comma sign has been added to the list of prohibited characters

### PR DESCRIPTION
**Pull request by salemgolemugoo**
I had issues with metric names which contain comma

Reading is failing with comma name:
fail

Successful:
successful

ChangeLog: collectd: Comma sign has been added to the list of prohibited characters

### CODE REVIEW

File: src/utils_format_graphite_test.c [hunk 1]
```diff
@@ -81,6 +81,11 @@ DEF_TEST(metric_name) {
           .type_instance = "test: \"hello\"",
           .want_name = "example@com.test-foo@@test@.single-test@@@hello@",
       },
+      {
+          .plugin_instance = "foo (test)",
+          .type_instance = "test: \"hello\"",
+          .want_name = "example,com.test-foo,,test,.single-test,,,hello,",
```
Comment (rjablonx) [inline src/utils_format_graphite_test.c]: This testcase is currently failing now with message:
_not ok 12 - got = "example@com.test-foo@@test@.single-test@@@hello@ 42 1480063672
", want "example,com.test-foo,,test,.single-test,,,hello, 42 1480063672
"._


I think this should be rewritten to something like this to actually test comma handling:
```
      /* test escaping comma */
      {
          .plugin_instance = "foo (test)",
          .type_instance = "test,123,",
          .want_name = "example@com.test-foo@@test@.single-test@123@",
      },
```

Comment (salemgolemugoo) [reply src/utils_format_graphite_test.c]: Thanks for help!

**Comment by salemgolemugoo** [comment]
@rjablonx please review

**Comment by salemgolemugoo** [comment]
Guys, who can approve this change?

**Comment by dago** [comment]
LGTM

**Pull request merged**